pkg/storaged/format-dialog.jsx

```diff
@@ -329,9 +338,13 @@ function format_dialog_internal(client, path, start, size, enable_dos_extended,
                 let new_crypto_options;
                 if (is_encrypted(vals)) {
                     let opts = [];
-                    if (vals.mount_options &&
-                        (!vals.mount_options.auto || vals.mount_options.never_auto)) {
-                        opts.push("noauto");
+                    if (vals.mount_options) {
+                        if (!vals.mount_options.auto || vals.mount_options.never_auto)
+                            opts.push("noauto");
+                        if (!vals.mount_options.fail)
+                            opts.push("nofail");
+                        if (vals.mount_options.netdev)
+                            opts.push("_netdev");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17888-20221108-113355-77d51bdb-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)